### PR TITLE
school table - make leaCode & estabCode non nullable integers

### PR DESCRIPTION
**Pull request by GuyHarwood**

### CODE REVIEW

**Comment by jon-shipley** [comment]
A quick scan reveals there are other places in the code that are eating estabCode as a string.

E.g. `  table.columns.add('estabCode', mssql.NVarChar(mssql.MAX), { nullable: true })`

This is from the school-import function.  I am sure there are others.

**Comment by GuyHarwood** [comment]
@jon-shipley yes i am working on these right now.  should have marked the PR as draft

**Comment by GuyHarwood** [comment]
@jon-shipley now ready for review

**Comment by GuyHarwood** [comment]
@jon-shipley please review when you have time

File: tslib/src/functions-throttled/school-import/data-access/school.data.service.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ export class SchoolDataService implements ISchoolDataService {
     const table = new mssql.Table('[mtc_admin].[school]')
     table.create = false
     table.columns.add('dfeNumber', mssql.Int, { nullable: false })
-    table.columns.add('estabCode', mssql.NVarChar(mssql.MAX), { nullable: true })
+    table.columns.add('estabCode', mssql.Int, { nullable: false })
     table.columns.add('leaCode', mssql.Int, { nullable: true })
```
Comment (jon-shipley) [inline tslib/src/functions-throttled/school-import/data-access/school.data.service.ts:38]: Shouldn't leaCode also be NOT NULL?

Comment (GuyHarwood) [reply tslib/src/functions-throttled/school-import/data-access/school.data.service.ts:38]: yep, fixed :)

**Review by jon-shipley — APPROVED** [review]
Thanks for this 💯

**Pull request merged**